docs/sources/flow/reference/stdlib/coalesce.md

```diff
@@ -0,0 +1,22 @@
+---
+aliases:
+- ../../configuration-language/standard-library/coalesce/
+title: coalesce
+---
+
+# coalesce
+
+`coalesce` takes any number of arguments and returns the first one that isn't null, an empty string, empty list, or an empty object.
+It is useful for obtaining a default value, such as if an environment variable isn't defined.
+The last argument is always returned, if there is no other non-empty or non-zero argument.
```

Comment:
I can't remember the link to the rule for this, but I've been told at some point to always make statements in the form of "If CONDITION, then STATEMENT" rather than "STATEMENT if CONDITION":

```suggestion
If no argument is non-empty or non-zero, the last argument is returned. 
```

Reply:
Found it: https://developers.google.com/style/sentence-structure

Reply:
Thanks, good to know.